charts/mgmt-agent/values.yaml

```diff
@@ -50,6 +50,16 @@ kubernetesCluster:
    name:
    # -- Kubernetes cluster namespace(s) to monitor. This can be a comma-separated list of namespaces or '*' to monitor all the namespaces
    namespace: '*'
+   # -- OCI namespace to push Kubernetes Monitoring metrics. The namespace should match the pattern '^[a-z][a-z0-9_]*[a-z0-9]$'. By default metrics will be pushed to 'mgmtagent_kubernetes_metrics'
+   monitoringNamespace:
+   # -- Provide the specific list of comma separated metric names for API server (/metrics) metrics to be collected.
+   overrideAllowMetricsAPIServer:
+   # -- Provide the specific list of comma separated metric names for agent computed metrics to be collected.
```

Comment:
Should we document about this specific metrics here ? also format ? may be a sample would helpful for each of these 4 types.

Reply:
these are the metrics from the k8s api, cAdvisor etc. the format is comma separated and the api which we invoke is mentioned in the comments.

Reply:
the comment was specific to overrideAllowMetricsAPIServer. It didn't mention API or how to get those metrics. If you give an example metric1, metric2 where the metric1 and metric2 represent some real metrics in those category, it may help.

Reply:
the API is there in comments i.e. /metrics
the standard way to interact with any k8s APIs is using `kubectl get` command.